### ISSUE TITLE
Switch Port - Lag / Glitch

### ISSUE THREAD

**Issue by NathanIfinity** [thestr4ng3r/chiaki#406]
Installed the latest atmo build and still having the exact same issue.. 

https://github.com/thestr4ng3r/chiaki/issues/405#issue-772551759

**Comment by H0neyBadger** [comment]
Hello, 
I guess you are running on 720p 60fps. 
I think the source of your problem might come from the network connection.
could you verify that you are using the best possible setup ?
ps4 <--(rj45)--> router <--(wifi 5ghz 802.11ac)--> nintendo switch 
(you can also disable the ps4 wifi to test the connection)

**Comment by NathanIfinity** [comment]
Yeah have been running on 720P 60fps since day one with no issue, Ps4 is connected with RJ45 and Switch is on 5Ghz band, I have a gigabit connexion (985 Dl / 975 Up) so don't think the issue is my network, and like I said it worked #1 since day one about a month or 2 ago, and just started glitching all of a sudden with no change in anything..
